### PR TITLE
Updated phantomjs location and version.

### PR DESCRIPTION
**Pull request by jjoos**
We're using this cookbook and it still pinned on an old version as default, so I thought let's update it for you.

Running this config with node variables already.

### CODE REVIEW

File: metadata.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@
 license 'Apache 2.0'
 description 'Installs/Configures phantomjs'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version '1.0.3'
+version '1.1.0'
```
Comment (sethvargo) [inline metadata.rb]: Please revert this change

**Comment by jjoos** [comment]
So i reverted the version number bump.

**Comment by jjoos** [comment]
@sethvargo Could you do a release, please?

**Comment by sethvargo** [comment]
I don't think this warrants a version bump since:
- This is just changing the default versions
- You can override these manually in your recipes
- It might break for existing users

**Comment by sciurus** [comment]
If this might break for existing users, shouldn't the version be bumped?

PhantomJS 1.9.7 has been out for 7 months now but the released cookbook's default is still 1.9.2.
